project/Dependencies.scala

```diff
@@ -4,7 +4,7 @@ object Dependencies {
   val Scala213 = "2.13.7"
   val ScalaVersions = Seq(Scala213)
 
-  val AkkaVersion = "2.6.17"
+  val AkkaVersion = "2.6.9"
```

Comment:
What's the rational for this specific version? Was there a breaking binary compatibility at this point? 

I note that other projects usually use any version there, as anyways the public-facing API should be binary compatible.

Reply:
yes so this is the minimal version that I found which lead to a successful build, I tried various.
this is something that can be shown in the documentation, as is the case here (I added this mention), so that it's clear for library users what's the minimal requirement. 
In [endless4s](https://endless4s.github.io/runtime.html) I use this constant from the build with paradox to automatically keep this version in sync in the docs. 
Notice `akka-http` itself uses this strategy because akka doesn't support mixed versions: https://doc.akka.io/docs/akka-http/current/compatibility-guidelines.html#compatibility-with-akka

That's unfortunate and a mystery to me why Akka isn't using e.g. an additional digit in the version number to reflect internal messaging incompatibilities in addition to the 3 semver digits instead of forbidding mixed versions via this runtime check (which is obviously unsafe).

Reply:
[akka-http-json](https://github.com/hseeberger/akka-http-json) is another project which [marks its akka dependencies as provided](https://github.com/hseeberger/akka-http-json/blob/master/build.sbt), however it's using indeed a very recent version of akka for compilation. However, as a library user I cannot infer from this whether it really needs that precise version of akka as a minimum for running, which I think is a drawback.

Reply:
@jchapuis `akka-actor` and `scaffeine` have a potencial conflict on `scala-java8-compat`. The minimum version we can support on `akka` needs to be `2.6.16` so that we don't have that issue.

```scala
[error] stack trace is suppressed; run last core / update for the full output
[error] (core / update) found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[error]
[error] 	* org.scala-lang.modules:scala-java8-compat_2.13:1.0.0 (early-semver) is selected over 0.9.0
[error] 	    +- com.github.blemale:scaffeine_2.13:5.1.1            (depends on 1.0.0)
[error] 	    +- com.typesafe.akka:akka-actor_2.13:2.6.15           (depends on 0.9.0)
```

Reply:
yes indeed not sure why I didn't spot it earlier 🤷 , setting to 2.6.16